doc/ref_arch/kubernetes/chapters/chapter04.rst

```diff
@@ -729,7 +703,7 @@ Architecture, they must be implemented according to the following specifications
      - N/A
    * - ra2.app.010
      - Node Feature Discovery (NFD)
-     - Workload descriptors must use the labels advertised by :cite:t:`k8s-nfd-sig` to indicate which node software of
+     - Workload descriptors should use the labels advertised by :cite:t:`k8s-nfd-sig` to indicate which node software or
```

Comment:
If this changes from "must" to "should": For workload to indicate which node software or hardware features they need, what other good descriptors can it use?
I would keep this and above one (Device Plugins) as "must".
Maybe both can be rewritten into "if workload needs... then it must...".

Reply:
Fixed in [30ee7d4](https://github.com/anuket-project/anuket-specifications/pull/3352/commits/30ee7d4f148513a980f65176cffc61ec074cdfcf)